daemons/controld/controld_schedulerd.c

```diff
@@ -300,8 +302,7 @@ controld_expect_sched_reply(char *ref)
     } else {
         controld_stop_sched_timer();
     }
-    free(fsa_pe_ref);
-    fsa_pe_ref = ref;
+    pcmk__str_update(&(controld_globals.fsa_pe_ref), ref);
```

Comment:
Before, we were stealing ref, but now you're duplicating it

Reply:
Whoops yeah, it's right there in the doxygen.